src/main.py

```diff
@@ -40,7 +43,18 @@ def main() -> None:
     """
 
     while True:
-        get_input()
+        #get imput 
```

Comment:
If the comment's use is to track that in this place, originally the get_input func was called, it is useless because the spelling was changed. Instead, use `# get_input()`
If it otherwise should comment on the functionality of the following code, it is not meaningful enough.